src/exporters/certHelpers.go

```diff
@@ -49,7 +51,13 @@ func secondsToExpiryFromCertAsBytes(certBytes []byte, password string) ([]certMe
 	if parsed {
 		return metrics, nil
 	}
-	return nil, fmt.Errorf("failed to parse as pem and pkcs12: %w", err)
+	// Parse as JKS
+	parsed, metrics, err = parseAsJKS(certBytes, password)
+	if parsed {
+		glog.Infof("JKS file parsing successful!")
```

Comment:
You could add this log for the other types (PEM, PKCS) to be consistent. Maybe lower the log level to debug/trace though cause there will be a lot of secrets.